src/sink/wavefront.rs

```diff
@@ -247,7 +251,13 @@ impl Wavefront {
 
         let mut tag_buf = String::with_capacity(1_024);
         let aggrs = mem::replace(&mut self.aggrs, Buckets::default());
-        for value in padding(aggrs.into_iter().filter(|x| x.is_zeroed()), self.bin_width) {
+        for value in
+            padding(aggrs.into_iter().filter(|x| x.is_zeroed()), self.bin_width)
```

Comment:
Sure thing. 

`aggrs.into_iter()` converts `aggrs` into a iterator, meaning `aggrs` is no longer a valid reference and we'll be given an iterator that owns the `Telemetry` formerly in `aggrs`. 

`.filter(|x| x.is_zeroed()` removes any zero valued `Telemetry` from the iterator stream. We do this in preparation for padding. 

`padding` is a function which returns a `Padding` iterator, one that adds zeros according to the procedure outlined here: https://github.com/postmates/cernan/issues/247#issuecomment-314837565 We previously removed all zero-values from the stream to avoid complicating the padding iterator. It's guaranteed that there are no zeros to have to cope with.